crates/primitives/src/kzg.rs

```diff
@@ -1,7 +1,11 @@
 mod env_settings;
 mod trusted_setup_points;
 
+#[cfg(feature = "c-kzg")]
 pub use c_kzg::KzgSettings;
+#[cfg(not(feature = "c-kzg"))]
+pub use kzg_rs::KzgSettings;
```

Comment:
It would be good to use https://docs.rs/cfg-if/latest/cfg_if/ 
And not having `c-kzg` does not mean that `kzg-rs` is enabled